### PR TITLE
Bitly - Get a Clicks Summary for a Bitlink

### PR DESCRIPTION
**Pull request by FlorentLvr**
This PR resolves https://github.com/jupyter-naas/awesome-notebooks/issues/1472

### CODE REVIEW

File: Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9599ccd-23d7-4488-837c-95af632141ce",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "clicks_summary = get_clicks_summary_for_bitlink(token, bitlink)\nclicks_summary",
```
Comment (srini047) [inline Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb:198]: Getting a  minor: JSONDecodeError

Screenshot 2023-02-24 at 7 46 38 PM

**Review by srini047 — COMMENTED** [review]
@FlorentLvr Fixing this might be enough for this template

File: Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb (hunk 1 above)
Comment (FlorentLvr) [reply Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb:198]: @srini047, do you try this code with one of your bitlink and setup your token? It's working on my side

Comment (srini047) [reply Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb:198]: @FlorentLvr Yes, I did try with my token & one of the bitly links that I have. I tried again. The same result.

Comment (FlorentLvr) [reply Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb:198]: @srini047, let's check together on Slack! I can't reproduce it, it's working on my side.

Comment (srini047) [reply Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb:198]: Sure @FlorentLvr

**Pull request merged**